pyvista/plotting/helpers.py

```diff
@@ -393,10 +393,10 @@ def plot_itk(mesh, color=None, scalars=None, opacity=1.0, smooth_shading=False):
         :func:`pyvista.wrap` can handle including NumPy arrays of XYZ
         points.
 
-    color : str or 3 item list, optional, defaults to white
+    color : color_like, optional, defaults to white
```

Comment:
See https://numpydoc.readthedocs.io/en/latest/format.html#parameters
```suggestion
    color : color_like, optional, default 'white'
```

Reply:
Good catch, actually should the 'optional' still be there then?

Reply:
We don't usually spell out defaults there, though. I don't remember seeing that elsewhere, do you know of other exampled @tkoyama010? Might be due to the fact that we switched to numpydoc later.

Reply:
> Good catch, actually should the 'optional' still be there then?

Oh. You are right. We should remove this.
```suggestion
    color : color_like, default 'white'
```
> We don't usually spell out defaults there, though. I don't remember seeing that elsewhere, do you know of other examples @tkoyama010? Might be due to the fact that we switched to numpydoc later.

Good point. Let's discuss in https://github.com/pyvista/pyvista/issues/2284 .

Reply:
Let's keep these changes for a separate PR then?